pyingest/tests/test_csgparsers.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
For now this is ok, but they should be put into the existing test_parsers.py file, because that contains all the tests for every other parser.